ci-operator/config/openshift/knative-client/openshift-knative-client-release-next__4.10.yaml

```diff
@@ -39,16 +39,14 @@ base_images:
     name: knative-nightly
     namespace: openshift
     tag: knative-serving-webhook
-  ocp_ubi-minimal_8:
-    name: ubi-minimal
-    namespace: ocp
-    tag: "8"
 binary_build_commands: |
   make install
   make build-cross
 build_root:
-  project_image:
-    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```

Comment:
will this in consequence mean, we get rid of the `openshift/ci-operator/build-image/Dockerfile` on the midstream?

If so, should we do this also for the serving/eventing repos?

Reply:
Yes, the idea is to replace the build-image everywhere ideally. If we agree on the current proposal. We should be able to control golang version more smoothly. E.g per Pierangelo's CI reconciler script. In theory the CI automation might be bumping golang on new version available, I haven't found out yet.

We don't get rid of the image entirely, since I've already added test-e2e-image to client's midstream. However, we separate test tools from Go version. 
https://github.com/openshift/knative-client/commit/7ff2766b97d2e37908a468de9a492fac6dea81ee